src/mrpro/data/_CsmData/_CsmData.py

```diff
@@ -93,3 +93,21 @@ def from_idata_walsh(
         csm_data = csm_fun(idata.data)
 
         return cls(header=idata.header, data=csm_data)
+
+    @classmethod
+    def from_idata_inati(cls, data: torch.Tensor, ks: int, power: int, padding_mode='circular') -> CsmData:
```

Comment:
if the method is called `from_idata...` I would expect that data is of type IData